### ISSUE TITLE
CTKEntry - Change background color when (state = 'readonly')

### ISSUE THREAD

**Issue by ivancasaof** [TomSchimansky/CustomTkinter#643]
How can I change the background color of CTKEntry when state is 'readonly'?

Thanks.

**Comment by Evolution0** [comment]
```python
if my_entry.state == "readonly":
    my_entry.configure(fg_color="black")
```

Valid colors, info, etc: https://github.com/TomSchimansky/CustomTkinter/wiki/Themes#colors

**Comment by TomSchimansky** [comment]
Fixed now with version 5.0.2. Thanks for reporting this.
